Source/ACE.Server/WorldObjects/Hooker.cs

```diff
@@ -44,11 +44,11 @@ public override ActivationResult CheckUseRequirements(WorldObject activator)
             if (!IsHooked(player, out var hook))
                 return new ActivationResult(new GameEventWeenieErrorWithString(player.Session, WeenieErrorWithString.ItemOnlyUsableOnHook, Name));
 
-            if (hook.HouseOwner == null || (!hook.House.OpenStatus && !hook.House.HasPermission(player)))
+            if (hook.HouseOwner == null || (!hook.House.RootHouse.OpenStatus && !hook.House.RootHouse.HasPermission(player)))
```

Comment:
does RootHouse need null checks here, in the rare chance that RootHouse failed to load?